### PR TITLE
chore(sidecar): cleanup + remove TODOs (continue)

### PR DESCRIPTION
**Pull request by merklefruit**
This PR addresses some small TODOs and cleans up the logic in the bolt manager pubkey verification.

### CODE REVIEW

File: bolt-sidecar/bin/sidecar.rs [hunk 1]
```diff
@@ -18,37 +18,22 @@ const BOLT: &str = r#"
 
 #[tokio::main]
 async fn main() -> eyre::Result<()> {
+    println!("{}", BOLT);
```
Comment (thedevbirb) [inline bolt-sidecar/bin/sidecar.rs:21]: Curious -- do you prefer displaying the logo as the first thing or after all startup checks passed successfully?

I like the idea of seeing the logo knowing I've done everything well :D but maybe that's just me

File: bolt-sidecar/src/primitives/delegation.rs [hunk 2]
```diff
@@ -57,15 +66,13 @@ impl SignableBLS for DelegationMessage {
 }
 
 /// read the delegations from disk if they exist and add them to the constraints client
-pub fn read_signed_delegations_from_file(
-    file_path: &PathBuf,
-) -> eyre::Result<Vec<SignedDelegation>> {
+pub fn read_signed_delegations_from_file(file_path: &PathBuf) -> Result<Vec<SignedDelegation>> {
```
Comment (thedevbirb) [inline bolt-sidecar/src/primitives/delegation.rs]: Can you keep `eyre::Result` as we discussed this morning?

**Review by thedevbirb — APPROVED** [review]

File: bolt-sidecar/bin/sidecar.rs (hunk 1 above)
Comment (merklefruit) [reply bolt-sidecar/bin/sidecar.rs:21]: I get that, but I didn't like having other logs before the logo, such as:

[info] telemetry initialized!
LOGO
[...] other logs

instead it should be:
LOGO
[...] all logs

Not a blocker ofc, let's just pick one :)

Comment (thedevbirb) [reply bolt-sidecar/bin/sidecar.rs:21]: nvm, let's merge this! We can modify it at a later time if we wish

**Pull request merged**